### DISCUSSION TITLE
Verweis Taste

### DISCUSSION THREAD

**Discussion by Diddeldab** [AppGround-io/bluetooth-keyboard-and-mouse-support#311] [Ideas]
Erst mal möchte ich sagen dass die App super ist und auch zuverlässig bei mir funktionniert...
Ein Feature das ich mir wünschen würde ist dass man in den Eigenen Layouts jeweils eine Taste erstellen kann um in ein anderes Layout zu springen (ohne dass man jedesmal über das Menü gehen muss um das Layout zu wechseln).
Vielen Dank MfG Marc

**Comment by AppGround-io** [comment]
Hallo Marc,

vielen Dank für dein positives Feedback und den interessanten Vorschlag!
Die Idee, eine Taste zum Hin- und Herwechseln zwischen Layouts zu
integrieren, finde ich sehr gut und werde auf jeden Fall prüfen, ob und wie
das machbar wäre.

Bis dahin: Schreib mir gerne, falls du weitere Fragen oder Ideen hast.

Beste Grüße,
Sebastian
